Automattic/crowdsignal-ui
# Store and send template on project save

## Summary

To track templates usage, we need to store the template selected by the user and send it to the backend when saving the project.

Depends on: D79559-code

Ref: c/k7PrVkyc-tr

## Test Instructions

* Apply backend changes to your sandbox
* Create a new project
* Choose one of the templates
* Save and publish the project
* Monitor the [Tracks Live](https://mc.a8c.com/tracks/live) page
* Both events `crowdsignal_project_create` and `crowdsignal_project_publish` should have `template` as one of their properties
